frontend/lib/pattern-helpers.js

```diff
@@ -11,4 +13,30 @@ function patternToOptions(pattern) {
   }
 }
 
-module.exports = { patternToOptions }
+// Removes regexp for named parameters.
+export function removeRegexpFromPattern(pattern) {
+  const tokens = pathToRegexp.parse(pattern)
+  const simplePattern = tokens
+    .map(token => {
+      if (typeof token === 'string') {
+        return token
+      } else {
+        const { delimiter, optional, repeat, name, pattern } = token
+        if (typeof name === 'number') {
+          return `${delimiter}(${pattern})`
+        } else {
+          let modifier = ''
+          if (optional && !repeat) {
+            modifier = '?'
+          } else if (!optional && repeat) {
+            modifier = '+'
+          } else if (optional && repeat) {
+            modifier = '*'
```

Comment:
I feel like in the future we may want to consider how we handle the *'s and ?'s. IIRC we have some patterns which have been added solely to avoid having the *'s show up in the frontend.